src/service/api.tsx

```diff
@@ -120,7 +137,7 @@ const api = {
 
   changeImgLinks(url: string, size: string) {
```

Comment:
it seems like this function does not change the link but return link (usually prefix "get" is used)

Reply:
thanks for pointing it out. I'll have a look at it